app/src/ui/diff/diff-syntax-mode.ts

```diff
@@ -166,7 +179,17 @@ export class DiffSyntaxMode {
         return null
       }
 
-      let result = getBaseDiffLineStyle(token)
+      const nextLine = stream.lookAhead(1)
+      const nextLineTokenIndex =
+        typeof nextLine === `string` ? nextLine[0] : undefined
```

Comment:
Not that it really matters in practice but we can just use single quotes here since we're not interpolating. 

```suggestion
        typeof nextLine === 'string' ? nextLine[0] : undefined
```